01-data-structures/02-stacks-and-queues/mystack/mystack_spec.rb

```diff
@@ -7,31 +7,57 @@
 
   describe "#push" do
     it "pushes an item on top of the stack" do
+      expect(stack.stack_length).to eq 0
       expect(stack.top).to eq nil
       stack.push("Rob")
       expect(stack.top).to eq "Rob"
+      expect(stack.stack_length).to eq 1
     end
 
     it "pushes an item on top of the stack" do
```

Comment:
you have the same verbiage twice.
These tests aren't structured properly

Start with an empty stack, test the correct properties
Then add your item, then check the correct properties

Each it should only test 1 thing